app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt

```diff
@@ -27,6 +27,8 @@ import org.fossasia.susi.ai.login.LoginActivity
 import org.fossasia.susi.ai.skills.SkillsActivity
 import org.fossasia.susi.ai.skills.settings.contract.ISettingsPresenter
 import org.fossasia.susi.ai.skills.settings.contract.ISettingsView
+import android.content.ComponentName
+
```

Comment:
Extra line

Reply:
@arundhati24 it was just so that code was easily readable.